spec/models/ability_spec.rb

```diff
@@ -69,6 +71,7 @@
               allow(user).to receive(:confirmed?).and_return(false)
             end
             it 'allows to see not hidden email address' do
+              pending "Fails. Unconfirmed user has no access"
```

Comment:
??? This spec is failing, but I am not sure if we even want it to pass for this conditions?

Reply:
Hmm. You're right. All an unconfirmed user should, in theory, be able to do is edit their email address.

Reply:
How about we keep it pending for now? I expect that it will solve itself in the upcoming Overhaul steps.

Reply:
Yes, I agree. Is there a way to mark this as a TODO? Or is the `pending` enough, do you reckon?

Reply:
I am currently collecting all the todo's, and will add them when we have finished the discussion in #991 . Don't want to add to much detail to that.

Reply:
Fair point!